piksi_tools/heartbeat.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Colin Beighley <colin@swift-nav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+from sbp.system import SBP_MSG_HEARTBEAT, MsgHeartbeat
+
+class Heartbeat(object):
```

Comment:
I agree on the link side, but I'm not sure how the context management line gets drawn between something like Heartbeat and Flash - the __exit__  function does the same thing in either case, removes the callbacks from the handler.